src/open_inwoner/openklant/clients.py

```diff
@@ -1,26 +1,246 @@
 import logging
-from typing import Optional
+from typing import List, Optional
```

Comment:
Just passing by: we recently removed all these occurrences to use the builtins instead (available since 3.9 and 3.10 for `str | None` syntax) in Open Forms using ruff to do it automatically. Related PRs:
- https://github.com/open-formulieren/open-forms/pull/3802
- https://github.com/open-formulieren/open-forms/pull/3818

Just have to be careful when converting `Union/Optional` to ` | `, it might fail at runtime e.g. when doing `"SomeForwardClass" | int` (in that case, this should be `"SomeForwardClass | int"`

Reply:
@pi-sigma Please let's stick to one standard and not individually start new styles and patterns; it'll make it even messier. We can change it in a big sweep of we want.

Reply:
This seems like something we should do in one go indeed